seed/analysis_pipelines/eui.py

```diff
@@ -219,15 +220,26 @@ def _run_analysis(self, meter_readings_by_analysis_property_view, analysis_id, p
     progress_data = ProgressData.from_key(progress_data_key)
     progress_data.step('Caclulating EUI')
 
+    Column.objects.get_or_create(
+        is_extra_data=True,
+        column_name="analysis_eui",
+        display_name="Analysis EUI",
+        organization=analysis.organization,
+        table_name='PropertyState',
+    )
+
     for analysis_property_view_id in meter_readings_by_analysis_property_view:
         analysis_property_view = AnalysisPropertyView.objects.get(id=analysis_property_view_id)
         property_state = PropertyState.objects.get(id=analysis_property_view.property_state.id)
+        eui = _calculate_eui(meter_readings_by_analysis_property_view[analysis_property_view_id], property_state.gross_floor_area.magnitude)
         analysis_property_view.parsed_results = {
-            'EUI': _calculate_eui(meter_readings_by_analysis_property_view[analysis_property_view_id], property_state.gross_floor_area.magnitude),
+            'EUI': eui,
             'Total Yearly Meter Reading': sum(meter_readings_by_analysis_property_view[analysis_property_view_id]),
             'Gross Floor Area': property_state.gross_floor_area.magnitude
         }
         analysis_property_view.save()
+        property_state.extra_data.update({'analysis_eui': eui})
+        property_state.save()
```

Comment:
I think this might not be the property state whose extra data we want to update. This changes the AnalysisPropertyView's PropertyState. The one we want to update is the PropertyView's PropertyState.

For reference, this is how things are modeled:
![Screen Shot 2021-09-01 at 2 25 55 PM](https://user-images.githubusercontent.com/18518728/131740080-791738c9-1ba7-435e-9750-d9ba78c24bd3.png)

The PropertyView is what's viewed in the inventory list, and is used for data quality checks. The AnalysisPropertyView is a snapshot (copy) of a PropertyView for usage in the analysis. So we need to update the PropertyView's PropertyState we took a snapshot of with the analysis result